### PR TITLE
Landing page fixes

### PR DESCRIPTION
**Pull request by SovereignAndrey**
## Explanation of the solution
First pass to clean up landing page code for RC-v2.1 deployment. 

## Instructions on making this work

- run `yarn` or `yarn install` to install npm dependencies
- run `yarn run test --watchAll` to verify all tests still pass
- (optional) run `yarn run build` to verify the build passes
- run `yarn start` to start the webapp
-

## UI changes for review

When major UI changes are introduced with a PR, please include links to URLS to compare or screenshots demonstrating the difference and notify on design changes

### CODE REVIEW

File: src/components/landing/BenefitsCarousel.jsx [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    setShowPrevButton(!swiper.isBeginning);
+    setShowNextButton(!swiper.isEnd);
+  };
+
+  const colors = ["#EDFCE2", "#EAE2FC", "#FCF6E2", "#EDF2FE"];
+
+  return (
+    <div className="lg:flex justify-between w-full  items-center p-16  hidden relative">
+      <div
+        className="swip-prev slider-arrow relative  left-[12.5%] translate-x-[-50%]  z-40   borderLine rounded-full shadow-md p-4"
```
Comment (misicnenad) [inline src/components/landing/BenefitsCarousel.jsx:32]:
```suggestion
        className="swip-prev slider-arrow relative left-[12.5%] -translate-x-1/2 z-40 select-none borderLine rounded-full shadow-md p-4"
```
nit: add `select-none` to stop highlighting text on the page when clicking the swiper arrow div multiple times

File: src/components/landing/BenefitsCarousel.jsx [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+  const [showPrevButton, setShowPrevButton] = useState(false);
+  const [showNextButton, setShowNextButton] = useState(true);
+
+  const handleSlideChange = (swiper) => {
+    setActiveIndex(swiper.activeIndex);
+    setShowPrevButton(!swiper.isBeginning);
+    setShowNextButton(!swiper.isEnd);
+  };
+
+  const colors = ["#EDFCE2", "#EAE2FC", "#FCF6E2", "#EDF2FE"];
```
Comment (misicnenad) [inline src/components/landing/BenefitsCarousel.jsx:27]: Colors can be moved to the parent component and be passed here as props - the same colors are duplicated and hardcoded in [Carousel.jsx](https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2899/files#diff-15dc7ce7ad4499b67dadfc25fd4552255997a98a5aac4d5cd546b15bfca8e084R22), so the parent can pass them as a prop to both components

File: src/components/landing/BlogCard.jsx [hunk 3]
```diff
@@ -1,42 +1,44 @@
...
-        <h4 className='text-[#0D283A] font-semibold w-full text-xl line-clamp-2  px-[24px]' dangerouslySetInnerHTML={{__html:(blog.title.rendered)}}/>
-        <p className='text-[#0D283A] text-[14px] lg:text-[16px] md:text-[16px] line-clamp-4 px-[24px]  tracking-[.8px]  text w-full  ' id='desc'
-            dangerouslySetInnerHTML={{ __html: sanitize(blog?.excerpt?.rendered) }} />
-        
-        <a href={`https://better.giving/${blog.slug}/`} className='self-end mt-auto  text-[#2D89C8] px-4  py-2 rounded-full  font-semibold uppercase'>Read More</a>
+import { DAPP_URL } from "constants/env";
+import DOMPurify from "dompurify";
+
+const BlogCard = ({ blog, imgUrl }) => {
+  const sanitize = (dirtyHTML) => {
```
Comment (misicnenad) [inline src/components/landing/BlogCard.jsx:5]: nit: can be moved outside the component

File: src/components/landing/AdminNavbar.jsx [hunk 4]
```diff
@@ -1,19 +1,48 @@
-import React, { useContext } from 'react'
-import { PageContext } from '../../pages/Home/state/PageState'
-import { UserContext } from '../../pages/Home/state/UserState'
+import React, { useContext } from "react";
+import { PageContext } from "../../pages/Home/state/PageState";
+import { UserContext } from "../../pages/Home/state/UserState";
 
 const AdminNavbar = () => {
```
Comment (misicnenad) [inline src/components/landing/AdminNavbar.jsx:5]: Can be removed, unused.

File: src/components/landing/Carousel.jsx [hunk 5]
```diff
@@ -0,0 +1,44 @@
+import React, { useEffect, useState } from "react";
+import "../../styles/landing/carousel.css";
+
+const Carousel = ({ slides }) => {
+  const [_right, setright] = useState(0);
```
Comment (misicnenad) [inline src/components/landing/Carousel.jsx:5]: Both `useState`s and both `useEffect`s can be removed, they do nothing

**Review by misicnenad — CHANGES_REQUESTED** [review]

File: src/components/landing/Carousel.jsx [hunk 6]
```diff
@@ -0,0 +1,44 @@
+import React, { useEffect, useState } from "react";
```
[the remark this answers is not in the document]
Comment (ap-justin) [reply src/components/landing/Carousel.jsx:1]: React needs to be imported on `.jsx files for test to pass

**Pull request merged**